### PR TITLE
Remove reference to non-existent warning

### PR DESCRIPTION
**Pull request by jfehrle**
Followup to #16291

Attn: @Zimmi48

### CODE REVIEW

**Review by Alizter — APPROVED** [review]

**Review by Zimmi48 — APPROVED** [review]
Looks like indeed, the PR that added this documentation (#8642) did not add the corresponding warning, so this was incorrect to begin with.

File: doc/sphinx/practical-tools/coq-commands.rst [hunk 1]
```diff
@@ -498,7 +498,6 @@ in a section.)
 
 When using ``-vos``, proofs in sections with :cmd:`Proof using` are skipped.  Proofs
 in sections without :cmd:`Proof using` are fully processed (much slower) and will raise a warning.
```
Comment (Zimmi48) [inline doc/sphinx/practical-tools/coq-commands.rst:500]: We should still check though if this claim "will raise a warning" is true or false. If it is false, then we should remove it as well.

Comment (jfehrle) [reply doc/sphinx/practical-tools/coq-commands.rst:500]: No, no warning.  My test:

`vos.v` is
```
Section A.
Theorem foo : True.
Proof.
apply I.
Qed.
End A.
```

```
$ coqc -vos vos.v
jfehrle@LAPTOP-LB5T3R83:/home/proj/coq2$ ls vos.*
vos.v  vos.vos
```

**Comment by jfehrle** [comment]
Updated.  I think ready to merge.

**Comment by jfehrle** [comment]
Whoops, now really updated.

**Comment by Alizter** [comment]
@coqbot merge now

**Pull request merged**